cockatrice/src/dlg_creategame.cpp

```diff
@@ -157,8 +175,41 @@ DlgCreateGame::DlgCreateGame(const ServerInfo_Game &gameInfo, const QMap<int, QS
     setWindowTitle(tr("Game information"));
 }
 
+void DlgCreateGame::actReset()
+{
+    descriptionEdit->setText("");
+    maxPlayersEdit->setValue(2);
+
+    passwordEdit->setText("");
+    onlyBuddiesCheckBox->setChecked(false);
+    onlyRegisteredCheckBox->setChecked(true);
+
+    spectatorsAllowedCheckBox->setChecked(true);
+    spectatorsNeedPasswordCheckBox->setChecked(false);
+    spectatorsCanTalkCheckBox->setChecked(false);
+    spectatorsSeeEverythingCheckBox->setChecked(false);
+
+    QMapIterator<int, QCheckBox *> gameTypeCheckBoxIterator(gameTypeCheckBoxes);
+    while (gameTypeCheckBoxIterator.hasNext()) {
+        gameTypeCheckBoxIterator.next();
+        gameTypeCheckBoxIterator.value()->setChecked(false);
+    }
+
+    descriptionEdit->setFocus();
+}
+
 void DlgCreateGame::actOK()
 {
+    settingsCache->setGameDescription(descriptionEdit->text());
+    settingsCache->setMaxPlayers(maxPlayersEdit->value());
+    settingsCache->setGamePassword(passwordEdit->text());
```

Comment:
But this isn't a password to anything really relevant. It's not an account or anything, it's for a simple game room. Just kick them if they weren't meant to be there. I'm up for a discussion on gitter if you'd like